avocado/plugins/list.py

```diff
@@ -101,10 +107,19 @@ def _display(self, test_matrix, stats):
             LOG_UI.debug(line)
 
         if self.args.verbose:
-            LOG_UI.debug("")
+            LOG_UI.info("")
+            LOG_UI.info("TEST TYPES SUMMARY")
+            LOG_UI.info("==================")
             for key in sorted(stats):
                 LOG_UI.info("%s: %s", key.upper(), stats[key])
 
+            if tag_stats:
```

Comment:
The difference is that we have a (more or less) defined set of test types, and that's why we show `0` counters for them.  With tags, it's a different matter.  We can only show the ones that are really given (there will never be a `0` entry for them).

So, that's why I think it makes sense to treat them differently.